### DISCUSSION TITLE
2006/07/14/%E5%8A%9E%E7%90%86%E6%8B%9B%E8%A1%8C%E4%BF%A1%E7%94%A8%E5%8D%A1%E7%9A%84%E7%97%9B%E8%8B%A6%E7%BB%8F%E5%8E%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#92] [Announcements]
# 2006/07/14/%E5%8A%9E%E7%90%86%E6%8B%9B%E8%A1%8C%E4%BF%A1%E7%94%A8%E5%8D%A1%E7%9A%84%E7%97%9B%E8%8B%A6%E7%BB%8F%E5%8E%86

我本来是有个招行普通信用卡的。大概两个月前，突然出现一笔 9 美元的消费，这不是我消费的，而且我当时又不在美国。所以我提出了申诉，招行处理申诉还是比较快的。两三天后就把钱退给了我。而且他们说为了防止类似情况再出现，会帮我把现在的卡销掉，给我免费换一个。我一直觉得我的信用卡额度太少了，用起来不方便，于是就要求他们直接把现在的卡销掉，不用给我新的了。我准备申请金卡。

https://blog.qizhen.xyz/2006/07/14/%E5%8A%9E%E7%90%86%E6%8B%9B%E8%A1%8C%E4%BF%A1%E7%94%A8%E5%8D%A1%E7%9A%84%E7%97%9B%E8%8B%A6%E7%BB%8F%E5%8E%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：


rongqiang
2006年10月5日 14:08 编辑
我一个朋友对招行的评价：态度好有屁用。:P

回复
Tian
2006年07月18日 16:44 编辑
晕。。。三年前招行叫我升级到金卡，两年前叫我升级到白金卡，我比较心疼300块的年费，不干。没准现在也升不到黄金卡了吧，哈哈

回复
yi
2006年07月16日 12:32 编辑
我不喜欢招行。服务不怎么好。
卡密码还要设置两个，不人性化

回复
Jack zhu
2006年07月15日 21:10 编辑
你好你的BLOG已被 博客无忧
 
http://www.ok591.cn
 
收录了能帮我们做好友情链接吗?谢谢

回复
Jack zhu
2006年07月15日 21:09 编辑
你好你的BLOG已被 博客无忧http://www.ok591.cn收录了能帮我们做好友情链接吗?谢谢

回复
☆┊☆┊ஐ♥you四哥小站:❉:‧:❉:‧:❉:
2006年07月15日 18:22 编辑
路过

回复
Yan
2006年07月15日 16:44 编辑
 
     你那些不良纪录，地球人都知道了啦……
 
     招行信用卡一直是印象最差的一个。
